src/UXClient/UXClient.ts

```diff
@@ -1,138 +1,60 @@
-        return new ScatterPlot(renderTarget);
-    }
-    
-    public BarChart(renderTarget){
-        return new GroupedBarChart(renderTarget);
-    }
-
-    public LineChart(renderTarget){
-        return new LineChart(renderTarget);
-    }
-
-    public AvailabilityChart(renderTarget){
-        return new AvailabilityChart(renderTarget);
-    }
-    
-    public Grid(renderTarget){
-        return new Grid(renderTarget);
-    }
-    
-    public Slider(renderTarget){
-        return new Slider(renderTarget);
-    }
-
-    public Hierarchy(renderTarget) {
-        return new Hierarchy(renderTarget);
-    }
-
-    public AggregateExpression(predicateObject: any, measureObject: any, measureTypes: Array<string>, searchSpan: any, 
-                                        splitByObject: any = null, colorOrOptionsObject: any, alias: string = '', 
-                                        contextMenu: Array<any> = []): any {
-        return new AggregateExpression(predicateObject, measureObject, measureTypes, searchSpan, splitByObject, colorOrOptionsObject, 
-                                        alias, contextMenu);
-    }
-
-    public TsqExpression(instanceObject: any, variableObject: any, searchSpan: any, 
-        colorOrOptionsObject: any, alias: string = '', contextMenu: Array<any> = []){
-            return new TsqExpression(instanceObject, variableObject, searchSpan, colorOrOptionsObject, alias, contextMenu);
-    }
-
-    public Heatmap(renderTarget) {
-        return new Heatmap(renderTarget);
-    }
-
-    public EventsTable(renderTarget) {
-        return new EventsTable(renderTarget);
-    }
-
-    public ModelSearch(renderTarget) {
-        return new ModelSearch(renderTarget);
-    }
-
-    public ModelAutocomplete(renderTarget) {
-        return new ModelAutocomplete(renderTarget);
-    }
-
-    public HierarchyNavigation(renderTarget) {
-        return new HierarchyNavigation(renderTarget);
-    }
-
-    public TimezonePicker(renderTarget) {
-        return new TimezonePicker(renderTarget);
-    }
-
-    public EllipsisMenu(renderTarget) {
-        return new EllipsisMenu(renderTarget);
-    }
-
-    public SingleDateTimePicker(renderTarget) {
-        return new SingleDateTimePicker(renderTarget);
-    }
-
-    public DateTimeButtonSingle (renderTarget) {
-        return new DateTimeButtonSingle(renderTarget);
-    } 
-
-    public DateTimeButtonRange (renderTarget) {
-        return new DateTimeButtonRange(renderTarget);
-    }
-
-    public ProcessGraphic(renderTarget) {
-        return new ProcessGraphic(renderTarget);
-    }
-
-    public PlaybackControls(renderTarget) {
-        return new PlaybackControls(renderTarget);
-    }
-
-    public ColorPicker(renderTarget) {
-        return new ColorPicker(renderTarget);
-    }
-
-    public GeoProcessGraphic(renderTarget) {
-        return new GeoProcessGraphic(renderTarget);
-    }
+    public DateTimePicker: typeof DateTimePicker = DateTimePicker;
```

Comment:
It might be a good idea to write a comment as to why we're exposing the components like this since it's not super obvious. Even I forgot why I went with this pattern instead of using namespace in my original branch ;). I believe it would have meant a breaking change for whoever consumed the client, but I don't remember 100%.

Reply:
Added this snippet.  How does that sound?  

Distributing typings broke the `new tsiClient.UxClient.LineChart(renderTarget)` syntax.  The functions we used before already returned newly constructed objects, so TypeScript complained, saying new wasn't allowed in that context.

So basically, this just allows the new syntax to work without typescript complaining.